dask/bag/text.py

```diff
@@ -103,6 +93,12 @@ def read_text(urlpath, blocksize=None, compression='infer',
         return from_delayed(blocks)
 
 
+def file_to_blocks(f):
+    with f as f:
+        for line in f:
+            yield line
```

Comment:
Having this as a task in the graph might cause problems, as generators aren't serializable.